pokerpoll/pokerpoll.py

```diff
@@ -62,18 +90,28 @@ def __init__(self, bot):
                 await self.bot.say(repr(reaction))
                 await self.bot.say(reaction)
                 await self.bot.say(user)
-                
 
-            good_boys = "\n".join(boy.mention for boy, status in react_hist.items() if status == True)
-            bad_boys = "\n".join(boy.mention for boy, status in react_hist.items() if status == False)
-            dash = "\n" + "-"*30 + "\n"
-            edit_string = f"@here poker time?\n\ni play to win kid{dash}{good_boys}\n\npoker's for nerds{dash}{bad_boys}"
+            good_boys = "\n".join(
+                boy.mention for boy, status in react_hist.items() if status)
+            bad_boys = "\n".join(
+                boy.mention for boy, status in react_hist.items() if not status)
+
+            edit_string = ("@here poker time?\n\n"
+                f"i play to win kid {self.dash}{good_boys}\n\n"
+                f"poker's for nerds {self.dash}{bad_boys}")
 
             await self.bot.edit_message(poll, new_content=edit_string)
-            
+
+            now = dt.now()
+            if now.hour >= playhour and now.minute >= playminute:
+                time_to_duel = True
+                if not playtime == "now":
+                    await self.bot.say(f"@here its time to duel{self.dash}{good_boys}")
+
+
 def check(reaction, check_user):
```

Comment:
i know this isn't your code, but this should be an inner function of `poker_poll` @JoeMitchellJones